homeassistant/components/hatchrest/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""The hatchrest integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+import async_timeout
+from bleak.exc import BleakError
```

Comment:
```suggestion
from bleak import BleakError
```
It's almost always better to import from the main module if the symbol is exported through it.

Reply:
Is it exported though? My linter (Pyright) tells me that it's not and suggests importing from `bleak.exc` instead. Maybe I have an older version on my branch.

Reply:
If you look at the other existing components for `BleakError` they seem to import it from the main module. That also worked fine when I tested it in local repl.